### PR TITLE
allow flexible schema and host deplyment, fix work on mac

### PR DESCRIPTION
**Pull request by tbicr**

### CODE REVIEW

File: Dockerfile-management [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:22.04
```
Comment (abitrolly) [inline Dockerfile-management:1]: Isn't it way to early?

Comment (tbicr) [reply Dockerfile-management:1]: it has osm2pgsql 1.6 which only works for me, no luck with the oldest versions in prev dist versions

File: docker-compose.yml [hunk 2]
```diff
@@ -19,15 +19,26 @@ version: "3.8"
 
 
 services:
+  management:
+    build:
+      context: .
+      dockerfile: Dockerfile-management
```
Comment (abitrolly) [inline docker-compose.yml:25]: What does this container really do? It doesn't spin any process.

File: docker-compose.yml [hunk 3]
```diff
@@ -19,15 +19,26 @@ version: "3.8"
...
+    build:
+      context: .
+      dockerfile: Dockerfile-webapp
     command: venv/bin/gunicorn index:application -w 4 --bind 0.0.0.0:8080
     depends_on:
       - postgis
       - redis
     env_file: .env
-    #ports:
-    #  - 8080:8080  # for dev purpose
```
Comment (abitrolly) [inline docker-compose.yml:30]: 127.0.0.1:8080:8080 should keep ports local without extra config files.

File: Dockerfile-management (hunk 1 above)
Comment (abitrolly) [reply Dockerfile-management:1]: Fair enough,

**Comment by abitrolly** [comment]
I need to run. Could review the PR that containe just for the "flexible schema", but this one is too large.

File: docker-compose.yml (hunk 2 above)
Comment (tbicr) [reply docker-compose.yml:25]: move a major management logic there, so it is also unavailable when tiles serve (similar to `tileserver`)

so in update.sh you can find major usage of it

File: docker-compose.yml (hunk 3 above)
Comment (tbicr) [reply docker-compose.yml:30]: it also will be available via `0.0.0.0` that I want to avoid, so will keep it as is

**Comment by tbicr** [comment]
tested locally, so will deploy changes, lets apply following improvements in separate pr

File: docker-compose.yml (hunk 3 above)
Comment (abitrolly) [reply docker-compose.yml:30]: No `127.0.0.1:8080:8080` specifically binds to local host interface.

**Comment by abitrolly** [comment]
https://openstreetmap.org is now fixed. Good job. :D

**Pull request merged**